src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/CmsRecipient.cs

```diff
@@ -15,6 +16,30 @@ public CmsRecipient(X509Certificate2 certificate)
         {
         }
 
+#if netstandard
```

Comment:
Do these two constructors need the internal/public treatment?  Or can they just entirely be wrapped in `#if !netstandard`?

Reply:
@bartonjs yeah, otherwise I was getting:

>Microsoft.DotNet.ApiCompat.targets(126,5): error : MembersMustExist : Member 'System.Security.Cryptography.Pkcs.CmsRecipient..ctor(System.Security.Cryptography.X509Certificates.X509Certificate2, System.Security.Cryptography.RSAEncryptionPadding)' does not exist in the reference but it does exist in the implementation.

Reply:
That seems... odd.  Since ApiCompat shouldn't be looking at `internal`.

Reply:
@bartonjs (removed previous comment since it had the wrong error message)

Yes, I think so otherwise I start getting compilation errors and I would need to put `#if` all other the place:

>Internal/Cryptography/PkcsHelpers.cs(181,41): error CS1729: 'CmsRecipient' does not contain a constructor that takes 3 arguments [/Users/user/Projects/3rdParty/corefx/src/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj]